examples/devices/lighty-actions-device/src/main/java/io/lighty/netconf/device/action/processors/ActionServiceDeviceProcessor.java

```diff
@@ -65,7 +67,7 @@ public QName getIdentifier() {
         final XmlElement fromDomElement = XmlElement.fromDomElement(requestXmlElement);
         final ActionDefinition actionDefinition = findActionInElement(fromDomElement);
 
-        if (actionDefinition.getQName().equals(Start.QNAME)) {
+        if (requireNonNull(actionDefinition).getQName().equals(Start.QNAME)) {
```

Comment:
Static import are good only in test class and constants class. But in this case, its makes the code difficult to understand. The main reason why this is difficult to read is that this method looks like belong to this class.